### DISCUSSION TITLE
Cant load GitHub on computer

### DISCUSSION THREAD

**Discussion by peachypeace** [community/community#49381] [Other Feature Feedback, Questions, & Ideas]
I can get in on phone but not on desktop every time it says err_add_invalid :/

**Comment by maahaMaya** [comment]


**Reply by peachypeace** [reply]
> yes there is GitHub Desktop you can download and use in the system.
> 
> 
> 
> https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/installing-github-desktop

I can't load the website therefore I can't log in ;-;

**Comment by peachypeace** [comment]
> The error "err_add_invalid" in GitHub usually occurs due to an invalid or misconfigured security certificate. Here are a few steps you can take to troubleshoot the issue:
> 
> 
> 
> Clear browser cache and cookies: Sometimes, clearing the browser cache and cookies can fix the issue. Try clearing the cache and cookies for your browser and then try accessing GitHub again.
> 
> 
> 
> Disable browser extensions: Certain browser extensions can interfere with the website's functionality. Try disabling all extensions and then try accessing GitHub again.
> 
> 
> 
> Check your system date and time: Make sure your computer's date and time are set correctly. An incorrect date and time can cause issues with security certificates.
> 
> 
> 
> Check your internet connection: Make sure you are connected to the internet and that your connection is stable.
> 
> 
> 
> Try a different browser: If you are still unable to access GitHub, try using a different browser.
> 
> 
> 
> If none of the above steps work, you may want to contact GitHub support for further assistance.

Thankyou it worked

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @peachypeace! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you.
